Core/src/org/sleuthkit/autopsy/datamodel/ImageNode.java

```diff
@@ -200,6 +204,27 @@ public String getItemType() {
         return getClass().getName();
     }
 
+    private Boolean checkSchemaVersion() {
+        String sqlStatement = "SELECT a.value creationMajorVersion, b.value creationMinorVersion FROM tsk_db_info_extended a, tsk_db_info_extended b " +
+                              " WHERE a.name = 'CREATION_SCHEMA_MAJOR_VERSION' and b.name = 'CREATION_SCHEMA_MINOR_VERSION';";
+        try (CaseDbQuery query = Case.getCurrentCaseThrows().getSleuthkitCase().executeQuery(sqlStatement);) {
+            ResultSet schemaVersion = query.getResultSet();
+            while (schemaVersion.next()) {
+                int creationMajorVersion = schemaVersion.getInt("creationMajorVersion");
+                int creationMinorVersion = schemaVersion.getInt("creationMinorVersion");
+                if ((creationMajorVersion == 8 && creationMinorVersion >= 3) || creationMajorVersion > 8) {
+                    schemaVersion.close();
+                    return true;
+                }
+            }
+            schemaVersion.close();
```

Comment:
SleuthkitCase.CaseDbQuery takes care of closing the ResultSet.

Reply:
Removed close.